docs/cpk.md

```diff
@@ -0,0 +1,139 @@
+# CPK methods
+
+## create
+```
+static async create(opts?: CPKConfig): Promise<CPK>
+```
+Static method that creates and initializes an instance of the CPK with the selected configuration parameters.
+
+## constructor
+```
+constructor(opts?: CPKConfig)
+```
+Creates a non-initialized instance of the CPK with the selected configuration parameters.
+
+## isProxyDeployed
+```
+async isProxyDeployed(): Promise<boolean>
+```
+Returns true or false if the Safe Proxy used by the CPK is already deployed (at least one transaction was made) or if it is not (no transactions made yet).
```

Comment:
What about a direct connection, either through WalletConnect or via the Safe Apps SDK, to a Safe with no transactions?